contrib/kubernetes/datahub/Chart.yaml

```diff
@@ -4,28 +4,7 @@ description: A Helm chart for LinkedIn DataHub
 type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 0.1.1
+version: 0.2.1
```

Comment:
If we are following SemVer, shouldn't it be upgraded to 0.2.0 from 0.1.1?

Reply:
Done!